includes/settings/class-wp-rest-api-log-settings.php

```diff
@@ -109,16 +109,37 @@ static private function current_tab() {
 		}
 
 
+		/**
+		 * Outputs the HTML for the settings tabs.
+		 *
+		 * @return void
+		 */
 		static private function plugin_options_tabs() {
 			$current_tab = self::current_tab();
 
-			echo '<h2>' . __( 'Settings' ) . ' &rsaquo; REST API Log</h2><h2 class="nav-tab-wrapper">';
+			echo '<h2>' . esc_html__( 'Settings' ) . ' &rsaquo; REST API Log</h2><h2 class="nav-tab-wrapper">';
```

Comment:
More of a toss-up, since it's the proper name of the plugin, and an acronym.  We'll cross that bridge when someone wants to do translations.